src/lessons/08_grid_layout/grid-layout.slides.html

```diff
@@ -0,0 +1,758 @@
+                # CSS
+                  .grid-container {
+                    grid-template-columns: [l1] 140px [l2] 150px [l3] auto [l4] 150px [l5] 140px [end];
+                    grid-template-rows: [r1-start] 25% [r1-end r2-start] 100px [r2-end r3-start] auto [r3-end];
+                  }
+
+                  .grid-item-1 {
+                    grid-column-start: 1;
+                    grid-column-end: span l4;
+                    grid-row-start: 2;
+                    grid-row-end: span 1;
+                  }
+      
+                # HTML
+                <div class="grid-container">
+                  <div class="grid-item-1">Item 1</div>
+                  <div class="grid-item-2">Item 2</div>
+                  <div class="grid-item-3">Item 3</div>
+                  <div class="grid-item-4">Item 4</div>
+                  <div class="grid-item-5">Item 5</div>
+                </div>
+        </code><br><div class="fragment grid-container grid-15">
+          <div class="grid-item-1">Item 1</div>
+          <div class="grid-item-2">Item 2</div>
+          <div class="grid-item-3">Item 3</div>
+          <div class="grid-item-4">Item 4</div>
+          <div class="grid-item-5">Item 5</div>
+      </div></pre>
+            </section>
+
+            <section>
+                <h3>Item Location - Shorthand</h3>
+                <code>grid-column, grid-row</code>
+                <br>
+                <pre><code class="formattedCode css">
+                .item {
+                  grid-column: &lt;start-line&gt; / &lt;end-line&gt; | &lt;start-line&gt; / span &lt;value&gt;;
+                  grid-row: &lt;start-line&gt; / &lt;end-line&gt; | &lt;start-line&gt; / span &lt;value&gt;;
+                }
+              </code></pre>
+            </section>
+
+            <section>
+                <h4>Example</h4>
+                <pre>
+            <code class="formattedCode css">
+                .grid-item-1 {
+                  <!-- grid-column-start: 1; -->
+                  <!-- grid-column-end: span l4; -->
+                  <!-- grid-row-start: 2; -->
+                  <!-- grid-row-end: span 1; -->
+                  grid-column: 1 / span l4;
+                  grid-row: 2 / span 1;
+                }
+      </code></pre>
+            </section>
+
+
+            <section>
+                <h3>Item Location - Even Shorter Shorthand</h3>
+                <code>grid-area</code>
+                <br>
+                <pre><code class="formattedCode css">
+                  .item {
+                    grid-area: &lt;name&gt; | &lt;row-start&gt; / &lt;column-start&gt; / &lt;row-end&gt; / &lt;column-end&gt;;
+                  }
+            </code></pre>
+            </section>
+
+            <section>
+                <h4>Example</h4>
+                <pre>
+          <code class="formattedCode css">
+            # as shorthand
+              .grid-item-1 {
+                <!-- grid-column: 1 / span l4; -->
```

Comment:
this comments are not showing on the slides. Also the formatting of the comment above is not right. not sure if is because of the html comment inside the css 🤔

Reply:
[Fixed](https://github.com/ReDI-Munich-Frontend/fed-munich-2021-spring/commit/1c4091a9316975c61acb733fbfdb941f98d3266a)